pytorch_lightning/plugins/precision/precision_plugin.py

```diff
@@ -32,41 +35,34 @@ class PrecisionPlugin(Plugin):
     EPSILON: float = 1e-6
     precision: Union[str, int] = 32
 
-    def __init__(self) -> None:
```

Comment:
no init needed anymore?

Reply:
I see some related edit bellow, but still, can you comment on it?

Reply:
Not needed, it would be empty